README.md

```diff
@@ -21,50 +39,112 @@ The shop has to provide a sphere client for all action this module
 
 ## Compiling
 Create **JAR** files:
-`sbt package`
+`./gradlew clean jar`
+
+Create **JAR** files and test:
+`./gradlew clean check`
+
+(**Recommended**) Full build with tests, but without install to maven local repo:
+`./gradlew clean build`
+
+Install to local maven repo:
+`./gradlew clean install`
 
-Create **JAR** and run tests:
-`sbt clean package test`
+Publish to Bintray:
+`./gradlew clean -Dbuild.version=$TRAVIS_TAG bintrayUpload`
+
+For more info about build and publish process see [BUILD](BUILD.md) documentation
 
 ## Integration into the shop:
 Take the dependencies from Maven central.
 
 ### Maven example
```

Comment:
i'd leave it, because it took some time for me to understand how to apply dependencies from JCenter to Maven project